### DISCUSSION TITLE
Deleting large video didn't remove file

### DISCUSSION THREAD

**Discussion by cheesebo** [photoprism/photoprism#3334] [Q&A]
Hi, 
I'm running Photoprism on Unraid. I've been working out the best way for me to upload my existing photo and video libraries and needed to clear out what I had uploaded. I noticed that after I had deleted all photos and videos, even though Photoprism shows as empty, there were 8 files left in the storage folders. These were all larger than 1GB.  
Is there an issue with deleting large files?

**Comment by lastzero** [comment]
How did you delete the files? I would assume these are transcoded versions. If so, you can delete them manually from the storage folder. You can also try the purge and cleanup CLI commands. If you are more specific, it's easier for community members to give you advice.
